platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/config/MerkleDbConfig.java

```diff
@@ -70,8 +70,7 @@
  */
 @ConfigData("merkleDb")
 public record MerkleDbConfig(
-        @Positive @ConfigProperty(defaultValue = "500000000") long maxNumOfKeys,
-        @Positive @ConfigProperty(defaultValue = "" + 4_000_000_000L) long size,
+        @Positive @ConfigProperty(defaultValue = "4000000000") long maxNumOfKeys,
```

Comment:
`maxNumOfKeys` previously was not "max", it was just some default value used as virtual map size hint in tests and benchmarks. I've no ideas where the old value, 500_000_000, came from.

Now this config is used as a real hard limit for the number of entries that can be stored in a single MerkleDb table.